raw-compare.js

```diff
@@ -409,6 +410,10 @@ function compareFiles(emisOutputFile, psOutputFile) {
 				outputReport = outputReport + resourceTypeCompareBuilder(resourceType, resourceTypeIndex, procedureRequestMatchFields, entriesByResource.emisEntries, entriesByResource.psEntries, allPsEntires)
 				break;
 			}
+			case "Specimen": {
```

Comment:
Do we need to add logic to lines 216, and 900?

Reply:
@adrianclay I've added some other changes based on your comment.